src/graphql/fragments/expenditures.graphql

```diff
@@ -15,6 +15,14 @@ fragment Expenditure on Expenditure {
       ...ExpenditureStage
     }
   }
+  actions(filter: { type: { eq: CREATE_EXPENDITURE } }) {
```

Comment:
maybe to keep consistent with how other actions are fetched we can rename this field to `creatingActions`?